app/editor/src/features/admin/sources/styled/DataSource.tsx

```diff
@@ -0,0 +1,4 @@
+import { Col } from 'components/flex/col';
+import styled from 'styled-components';
+
+export const DataSource = styled(Col)``;
```

Comment:
Guessing there is going to be styling down the line here

Reply:
As a habit I'm making styled components for most of our features as they will most likely need some styling at some point.